components/web_discovery/common/features.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/web_discovery/common/features.h"
+
+namespace web_discovery::features {
+
+BASE_FEATURE(kWebDiscoveryNative,
+             "BraveWebDiscoveryNative",
```

Comment:
variable and feature name should be in sync. Either `kBraveWebDiscoveryNative` or `"WebDiscoveryNative"`.